default.py

```diff
@@ -491,5 +552,7 @@ def add_dir(name, url, mode, iconimage, info, page=0):
     categories()
 elif MODE == 1:
     index(NAME, URL, PAGE)
+elif MODE == 2:
+    categories(show_archived=True)
```

Comment:
Yea it was kind of daunting initially, but look at the docs, that's the way. I could see a very complex addon having a ridiculous amount of if/elif based on Mode ha